client/my-sites/site-settings/site-owner-transfer/site-owner-user-search.tsx

```diff
@@ -22,6 +22,10 @@ const TextControlContainer = styled.div( {
 	},
 } );
 
+const ButtonStyled = styled( Button )`
+	margin-bottom: 1em;
+`;
```

Comment:
```suggestion
const ButtonStyled = styled( Button )({
	marginBottom: '1em'
})
```

Reply:
Sure @sejas ! I am usually seeing both being used so I am newer sure which one I should be using. I remember saying that one of these method is older and one is newer, is that right?

Reply:
There is a discussion here p1660141381467549-slack-C0347E545HR.
The TLDR is that with JS objects you get autocomplete, type checker and error linter. The probability of introducing a typo in a CSS attribute is zero.
With string templates we don't get any of those benefits.